lib/devise_security_extension/models/password_archivable.rb

```diff
@@ -16,7 +16,7 @@ def validate_password_archive
 
       # validate is the password used in the past
       def password_archive_included?
-        unless deny_old_passwords.is_a? Fixnum
+        unless deny_old_passwords.is_a? 1.class
```

Comment:
Why not use `Integer` here? Even in Ruby < 2.4:

```ruby
1.class #=> Fixnum
1.is_a? Integer #=> true
```

Reply:
The maintainers of this repo seemed to have disappeared, so a few of us are maintaining a fork at https://github.com/devise-security/devise-security. Feel free to make a PR with these changes over there.